packages/sit-onyx/src/components/OnyxFormElement/OnyxFormElement.vue

```diff
@@ -49,29 +53,26 @@ defineSlots<{
     </div>
     <slot :id="props.id"></slot>
     <div class="onyx-form-element__footer onyx-text--small">
-      <span v-if="errorMessages" class="onyx-form-element__error-message onyx-truncation-ellipsis">
-        {{ errorMessages.shortMessage }}
-      </span>
-      <OnyxInfoTooltip
-        v-if="errorMessages?.longMessage"
-        class="onyx-form-element__error-tooltip"
-        color="danger"
-        position="bottom"
-        :label="t('tooltip.error')"
-        open="hover"
-        :text="errorMessages.longMessage"
-      />
-
-      <span v-if="props.message" class="onyx-form-element__message onyx-truncation-ellipsis">
-        {{ props.message }}
+      <span class="onyx-form-element__footer-messages">
+        <FormMessage
+          v-if="props.errorMessages"
+          class="onyx-form-element__error-message"
+          :messages="props.errorMessages"
+          type="danger"
+        />
+        <FormMessage
+          v-if="props.successMessages"
+          class="onyx-form-element__success-message"
+          :messages="props.successMessages"
+          type="success"
+        />
+        <FormMessage
+          v-if="props.message"
+          class="onyx-form-element__message"
+          :messages="props.message"
+          type="neutral"
+        />
```

Comment:
Please add a test to ensure that the correct message is shown for every possible combination